_posts/2023-02-22-gsoc2023.md

```diff
@@ -0,0 +1,127 @@
+| 5       | [Transport property calculations]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-5-transport-property-calculations) | medium     | 350 hours   |  write analysis code to calculate physical transport properties | Python, Physics/Mathematics      | @orionarcher, @hmacdope |
+| 6       | [Implementation of parallel analysis framework]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-6-parallel-analysis) | hard     | 350 hours   |  implement parallel framework in MDAnalysis| Python, Parallel Programming, Molecular Modelling      | @yuxuanzhuang, @orbeckst, @RMeli |
+
+
+
+## Information for prospective GSoC Contributors
+
+You must meet our [own requirements] if you want to be a GSoC
+Contributor with MDAnalysis this year (read all the docs behind these
+links!). You must also meet the [eligibility criteria]. Our [GSoC
+FAQ][] collects common questions from applicants.
+
+The MDAnalysis community values diversity and is committed to providing a
+productive, harassment-free environment to every member. Our [Code of Conduct]
+explains the values that we as a community uphold. *Every member (and every
+GSoC Contributor) agrees to follow the [Code of Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+### Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+### Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us your github handle, what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+### Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of MDAnalysis* in
+order to be eligible, i.e., you must demonstrate that you have been seriously
+engaged with the MDAnalysis project.  We have a list of [easy bugs][] and
+suggested [GSOC Starter issues][GSOC Starter] to work on in our issue tracker
+on GitHub. *We only accept one [GSOC Starter issue][GSOC Starter] per
+applicant* so that everybody gets a chance. If you want to dive deeper, we
+encourage you to tackle some of the other issues in our issue tracker.
+
+We also appreciate contributions which add more tests or update/improve our documentation.
+
+We recommend you start your application by working on an issue. It will give
+you a better understanding of MDAnalysis as a project and improve the quality
+of your application.
+
+To start developing for MDAnalysis have a look at our
+guide on [contributing to MDAnalysis][dev-guide] and write to us on
+the [mailing list][] if you have more questions about setting up a
```

Comment:
Which mailing list should this be? Still the GSoC one or the Developer one?

Reply:
I switched it from devs to gsoc to make sure that gsoc candidates only need to come to one place.

Reply:
Ok, I just wanted to double check this was intended.